packages/swarm-styles/lib/spacing.css

```diff
@@ -1,3 +1,17 @@
 :root {
     --space: 16px;
-}
\ No newline at end of file
+}
+
+/* Medium breakpoint overrides */
+@media only screen and (min-width: 640px) {
+  :root {
+      --responsive-space: 20px;
```

Comment:
Should these be using `--size-space-large-default`?

Reply:
@sadafie yes, this is how and where the responsive spaces would be defined.